MANIFEST.in

```diff
@@ -1,5 +1,4 @@
 include README.rst
 include LICENSE
 include Makefile
-include examples/*.yaml
```

Comment:
Yes, the files do not exist anymore. I'm wondering why am I not able to reproduce the failure by:

```
make develop 2>&1 | grep "no file"
```
Anyway it looks good and removes non-existing include so ack.